Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -236,6 +236,14 @@ class BrowserViewController: UIViewController {
         
         setupRewardsObservers()
         
+        if let rewards = rewards, !Preferences.Rewards.checkedPreviousCycleForAdsViewing.value {
```

Comment:
Ref #1604, the `let rewards = rewards` will have to be removed in that PR

Reply:
I'll rebase and merge the NO_REWARDS PR after this one is merged